glide.yaml

```diff
@@ -200,7 +200,7 @@ import:
 - package: github.com/russross/blackfriday
   version: 300106c228d52c8941d4b3de6054a6062a86dda3
 - package: github.com/shirou/gopsutil
-  version: v2.0.0
+  version: 6e9b0bbb1ba299ce4eb978a6882f48c0080a7858
```

Comment:
We normally try to pick released versions instead of a commit hash. That helps with rkt packaging in distributions like Debian.

Could you pick gopsutil v2.1 instead? It seems to be more recent than 6e9b0bbb1ba299ce4eb978a6882f48c0080a7858